roles/oslat/templates/oslatjob.yaml

```diff
@@ -15,7 +15,7 @@ spec:
 {% if workload_args.runtime_class is defined %}
       runtimeClassName: "{{ workload_args.runtime_class }}"
 {% endif %}
-{% if workload_args.nodeselector is defined %}
+{% if workload_args.node_selector is defined %}
```

Comment:
Same as above but for `oslat`:
https://github.com/cloud-bulldozer/benchmark-operator/blob/master/roles/oslat/templates/oslatjob.yaml#L24

Reply:
see above.